filter_mitm_logs.py

```
@@ -82,35 +90,42 @@ def filter_jsonl_file(input_file, output_file, url):
+                                    break
+
+                    line_number += 1
                 except KeyError as e:
-                    logging.error(f"Skipping line: {e}")
+                    logging.error(f"Skipping line {line_number}: {e}")
+                    line_number += 1
 
         logging.info(f"Filtered output written to: {output_file}")
     except FileNotFoundError:
-        logging.error(f"Error: The file {input_file} was not found.")
+        logging.error(f"Error: {e}")
```

Ruff (0.8.2)

Undefined name `e`

(F821)